api/validate-api.http

```diff
@@ -0,0 +1,46 @@
+### Save Bundle
```

Comment:
is there a way for us to automate this as a post-deployment validation via pipelines?

Reply:
actually, if we keep it this way, can you add a little comment describing how to utilize this :)

Reply:
I plan to add a link or verbiage in the README on how to use the tool.

Reply:
This is more for dev time checking.  We would need to implement some of these as integration tests for post build integration checks.

Reply:
Yup, this is mainly for new developers to validate that they have the app set up correctly